examples/mnist_transfer_cnn.py

```diff
@@ -91,13 +91,13 @@ def train_model(model, train, test, num_classes):
 
 # define two groups of layers: feature (convolutions) and classification (dense)
 feature_layers = [
-    Convolution2D(filterss, kernel_size, kernel_size,
```

Comment:
Please also rename `filterss` to `filters`, that was obviously a glitch with global search-and-replace. And rename `Convolution2D` to `Conv2D`.